src/goodbase/goodbase_cli.py

```diff
@@ -544,9 +553,19 @@ def dependencies(binder: inject.Binder) -> None:
         revision = orchestrator.checkout_good_base(evg_project, criteria)
 
         if revision:
-            click.echo(click.style(f"Found revision: {revision.revision}", fg="green"))
+            revision_dict = {}
+            revision_dict["mongo"] = revision.revision
             for module_name, module_revision in revision.module_revisions.items():
-                click.echo(click.style(f"\t{module_name}: {module_revision}", fg="green"))
+                revision_dict[module_name] = module_revision
+
+            if output_format == "yaml":
+                click.echo(click.style(yaml.dump(revision_dict, sort_keys=False), fg="green"))
```

Comment:
Do you want to keep the style for the `yaml` and `json` formats? It seems like they would be meant to be consumed by scripts and the style would end up just be terminal characters that the scripts would need to ignore. We might want to only apply the stying if the plaintext option is used.

Reply:
Good point. Addressed in 650b276